[2025-03-03T12:22:18.091-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-03T12:22:18.131-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: EtlWeather.SendMessage scheduled__2025-03-02T00:00:00+00:00 [queued]>
[2025-03-03T12:22:18.157-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: EtlWeather.SendMessage scheduled__2025-03-02T00:00:00+00:00 [queued]>
[2025-03-03T12:22:18.158-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-03T12:22:18.206-0600] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): SendMessage> on 2025-03-02 00:00:00+00:00
[2025-03-03T12:22:18.215-0600] {standard_task_runner.py:72} INFO - Started process 4550 to run task
[2025-03-03T12:22:18.223-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'EtlWeather', 'SendMessage', 'scheduled__2025-03-02T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmp5nkzplms']
[2025-03-03T12:22:18.229-0600] {standard_task_runner.py:105} INFO - Job 60: Subtask SendMessage
[2025-03-03T12:22:18.351-0600] {task_command.py:467} INFO - Running <TaskInstance: EtlWeather.SendMessage scheduled__2025-03-02T00:00:00+00:00 [running]> on host IMNOTROY.
[2025-03-03T12:22:18.635-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='EtlWeather' AIRFLOW_CTX_TASK_ID='SendMessage' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T00:00:00+00:00'
[2025-03-03T12:22:18.637-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-03T12:26:02.864-0600] {python.py:240} INFO - Done. Returned value was: Done!
[2025-03-03T12:26:02.980-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-03T12:26:02.982-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=EtlWeather, task_id=SendMessage, run_id=scheduled__2025-03-02T00:00:00+00:00, execution_date=20250302T000000, start_date=20250303T182218, end_date=20250303T182602
[2025-03-03T12:26:03.088-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-03T12:26:03.143-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-03T12:26:03.146-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
